app/webhook.go

```diff
@@ -133,7 +133,13 @@ func (a *App) TriggerWebhook(payload *model.OutgoingWebhookPayload, hook *model.
 						if len(webhookResp.Attachments) > 0 {
 							webhookResp.Props["attachments"] = webhookResp.Attachments
 						}
+						if a.Config().ServiceSettings.EnablePostUsernameOverride && hook.Username != "" {
```

Comment:
Hi @lieut-data I couldn't understand this point completely..  Kindly clarify.. 

I understood it as follows, if the web hook responds with a non empty username and icon_url, that should be taken instead of the configured ones. 

Am I get it right ?

Reply:
Yep, you got it @pradeepmurugesan! Right now, it looks like the configured ones apply unconditionally, but this should behave similarly to the incoming webhooks.

Reply:
done..